modules/gateway/synchronize.go

```diff
@@ -24,7 +21,7 @@ func (g *Gateway) Synchronize() (err error) {
 	if err != nil {
 		return
 	}
-	go g.synchronize(peer)
+	g.synchronize(peer)
```

Comment:
We want Synchronize() to be a blocking function?

I thought we wanted people to be able to use the interface while startup was happening.